### PR TITLE
cleaned up UI and added image viewer window

### PR DESCRIPTION
**Pull request by luiskuhn**
Screenshot 2021-06-09 at 16 36 07

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,22 @@ Changelog
...
+------------------
+
+**Added**
+
+* Added a window for the image viewer
+
+**Fixed**
+
+* Cleaned up metadata window
+* Replaced the image size and timepoints columns for a single column
```
Comment (jenniferboedker) [inline CHANGELOG.rst:17]: Maybe link the respective PRs or here

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,22 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.1.1 (2021-06-09)
+------------------
+
+**Added**
+
+* Added a window for the image viewer
```
Comment (jenniferboedker) [inline CHANGELOG.rst:12]: when adding a feature I would expect it to be not only a patch release (then it should be 1.2.0), what do you think?

File: CHANGELOG.rst (hunk 1 above)
Comment (luiskuhn) [reply CHANGELOG.rst:17]: will do

File: CHANGELOG.rst (hunk 2 above)
Comment (luiskuhn) [reply CHANGELOG.rst:12]: makes sense

**Pull request merged**